O/OpenLDAPClient/build_tarballs.jl

```diff
@@ -28,7 +28,29 @@ if [[ "${target}" == *-mingw* ]]; then
 	export LDFLAGS="-lpcreposix-0 -L${prefix}/bin"
 fi
 
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-slapd --without-yielding_select --disable-static --enable-shared --enable-modules=yes --enable-hdb=no --enable-bdb=no
+if [[ "${target}" != *-freebsd* ]]; then
+    # The mac build tries to pick up libraries from system root, but our build
+    # of OpenSSL seems to be incomplete (it's missing some important symbols),
+    # so let's keep this file for FreeBSD until we get reports it doesn't work.
+    rm -f /opt/${target}/${target}/sys-root/usr/lib/libcrypto.*
+    rm -f /opt/${target}/${target}/sys-root/usr/lib/libssl.*
+    rm -f /opt/${target}/${target}/sys-root/usr/lib/libsasl2.*
+    rm -f /lib/libcrypto.so*
+    rm -f /usr/lib/libcrypto.so*
+    rm -f /lib/libssl.so*
+    rm -f /usr/lib/libssl.so*
```

Comment:
I can just feel your zealous fury as you delete every OpenSSL-related library on your system.  😂 

Those are host SSL libraries; you shouldn't delete these or else you won't be able to do stuff like `curl`.  :P Only the `/opt/${target}` libraries will be used by the compiler, because that's the only place that gets searched.  Even if it did search `/lib`, it would not link in these libraries because they won't match the compiler's target (except when `${target}` == `x86_64-linux-musl`).

Reply:
In the OpenSSH builder for `x86_64-linux-musl`, the build system was picking up the system OpenSSL.

> you shouldn't delete these or else you won't be able to do stuff like curl.

Do we want to run `curl` during a build? :wink: